core/model/modx/processors/search/search.class.php

```diff
@@ -186,6 +188,7 @@ public function searchResources()
                 'type' => $type,
                 'class' => $record->get('class_key'),
                 'type_label' => $typeLabel,
+                'icon' => $record->get('icon')?(str_replace('icon-','',$record->get('icon'))):false,
```

Comment:
Can be slightly simplified to 

````
'icon' => $record->get('icon') ? str_replace('icon-', '', $record->get('icon')) : false,
````

or perhaps put above the `$this->results[] = array(` where it only has to get the icon into a variable once, and then manipulates that?